lib/elixir/lib/calendar/date.ex

```diff
@@ -173,6 +173,72 @@ defmodule Date do
     }
   end
 
+  @doc """
+  Returns the current date in the provided time zone.
+
+  By default, it uses the default time_zone returned by
```

Comment:
```suggestion
  By default, it uses the default time zone database returned by
```

(if this text change sounds good, need to change DateTime.now/2 too)